### PR TITLE
docs: update DORA manuals in main and v0.20

### PR DESCRIPTION
**Pull request by Startrekzky**
### ⚠️ &nbsp;&nbsp;Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have `npm run build` and `npm run serve` locally before submitting this PR
- [x] I have read through the [Contributing](https://devlake.apache.org/community/) Documentation

# Summary
Update DORA manuals in main and v0.20 based on the latest workflow and UI.

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: docs/DORA.md [hunk 1]
```diff
@@ -164,27 +150,42 @@ workflows:
...
-    \"commit_sha\":\"$CIRCLE_SHA1\",
-    \"repo_url\":\"$CIRCLE_REPOSITORY_URL\",
-    \"start_time\":\"$start_time\"
-  }'
-```
+2.5 Run the CircleCI pipeline. Visit DevLake's DB to check if the deployments have been successfully pushed to DevLake. The deployments will appear in table [cicd_deployments](DataModels/DevLakeDomainLayerSchema.md#cicd_deployments) and [cicd_deployment_commits](DataModels/DevLakeDomainLayerSchema.md#cicd_deployment_commits) in DevLake's database.
+
+  ![CircleCI-screenshot](https://i.imgur.com/IyneAMn.png)
+
+  ![webhook-query](/img/ConfigUI/webhook-query-deployments.png)
```
Comment (d4x1) [inline docs/DORA.md:158]: Is it correct?  I cannot see the pic on preview mode.

image
image

File: docs/DORA.md [hunk 2]
```diff
@@ -164,27 +150,42 @@ workflows:
...
-```
-curl https://sample-url.com/api/plugins/webhook/2/deployments -X 'POST' -u 'username:password' -d '{
-    \"commit_sha\":\"$CIRCLE_SHA1\",
-    \"repo_url\":\"$CIRCLE_REPOSITORY_URL\",
-    \"start_time\":\"$start_time\"
-  }'
-```
+2.5 Run the CircleCI pipeline. Visit DevLake's DB to check if the deployments have been successfully pushed to DevLake. The deployments will appear in table [cicd_deployments](DataModels/DevLakeDomainLayerSchema.md#cicd_deployments) and [cicd_deployment_commits](DataModels/DevLakeDomainLayerSchema.md#cicd_deployment_commits) in DevLake's database.
+
+  ![CircleCI-screenshot](https://i.imgur.com/IyneAMn.png)
```
Comment (d4x1) [inline docs/DORA.md]: Is it a mistake ? I think we shouldn's depend on imgur.com.

Comment (Startrekzky) [reply docs/DORA.md]: @d4x1 We shouldn't rely on imgur.com. I'll get them updated.

File: docs/DORA.md (hunk 1 above)
Comment (Startrekzky) [reply docs/DORA.md:158]: @d4x1  It should be a network problem. Never mind, I'll modify them to local files.

**Review by d4x1 — APPROVED** [review]

**Pull request merged**